landez/sources.py

```diff
@@ -211,6 +238,9 @@ def __init__(self, url, layers, headers=None, tilesize=None, **kwargs):
         if parse_version(self.wmsParams['version']) >= parse_version('1.3'):
             projectionKey = 'crs'
         self.wmsParams[projectionKey] = GoogleProjection.NAME
+        self.lock = threading.Lock()
```

Comment:
Indeed.
I'm having a bit of trouble to keep track of what is private and what is public as I haven't done python for quite a time and have done mostly c++/c# where it is clearly defined which are which.